platform/packaging/build/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM docker.io/library/debian:12
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Phase 1: Install free components.
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
+    python3-dev python3-numpy-dev swig python3-virtualenv \
+    cmake git gpg wget
+
+# Phase 2: Install non-free components.
+RUN sed -i -e 's/main/main contrib non-free/g' /etc/apt/sources.list.d/debian.sources
```

Comment:
For the future, I am not a fan of interpolation in all shapes and sizes. Running `sed` on files is the worst kind of interpolation. It has the appearance of a magic number variable.

A better approach is a drop-in file, which I can get behind ;-)

Reply:
Ha, yeah, I come from Gentoo and Nix, so this sort of thing is how I think about enabling `non-free`. We can change this.